test/notification.go

```diff
@@ -0,0 +1,17 @@
+package test
+
+import (
+	"context"
+
+	"github.com/fabric8-services/fabric8-auth/notification"
+)
+
+// NotificationChannel is a simple Sender impl that records the notifications for later verification
+type NotificationChannel struct {
+	Messages []notification.Message
```

Comment:
Doesn't look like you use these messages in any tests?

Reply:
I'm using `test.NotificationChannel` but not the `Messages` yet - been trying to figure out some pointer indirection challenges with `Channel`.

Initial idea was(is) to inspect the `Messages` array in tests.